### PR TITLE
chore(webapp): upgrade monorepo dependencies

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:
Typeahead style has been updated according to Bootstrap theme

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/156/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/156/)

File: dataprep-webapp/src/app/components/widgets/typeahead/_typeahead.scss [hunk 1]
```diff
@@ -50,45 +46,38 @@ typeahead {
...
+	button {
 		&.inverse {
 			position: absolute;
 			z-index: $typeahead-z-index;
 
+			&:hover,
+			&:active,
+			&:focus,
+			&:visited {
+				background-color: transparent !important;
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets/typeahead/_typeahead.scss]: !important?

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/165/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/165/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/166/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/166/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/168/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/168/)

**Pull request merged**